modules/routes/calendar/calendar_event_route.py

```diff
@@ -0,0 +1,50 @@
+from flask import request, jsonify
+from app import app
+from modules.models.account import AccountType
+from modules.routes.calendar.calendar_event_schema import (
+    CalendarEventSchema,
+
+)
+from modules.utils.db import db
+from modules.utils.route_utils import authenticate, fetch_session, protected_route
+
+@app.route("/calendar", methods=["POST"])
```

Comment:
The course_id should be in the route. Remove it from payload schema.

```suggestion
@app.route("/calendar/<str:course_id>", methods=["POST"])
```